config/Configuration.java

```diff
@@ -0,0 +1,7 @@
+package pl.futurecollars.invoicing.config;
+
+public class Configuration {
+
+  public static final String INVOICES_PATH = "invoices.txt";
```

Comment:
Jeśli w pliku zapisujesz dane w formacie json, to dajmy mu odpowiednie rozszerzenie (json).